honeybadgermpc/hbavss.py

```diff
@@ -299,24 +324,19 @@ def _get_dealer_msg(self, values):
         logger.debug("[%d] Starting Batch AVSS. Id: %s, Dealer Id: %d, Client Mode: %s",
                      self.my_id, avss_id, dealer_id, client_mode)
 
-        broadcast_msg = None if self.my_id != dealer_id else self._get_dealer_msg(values)
+        dispersal_msg_list = None if self.my_id != dealer_id else self._get_dealer_msg(
+            values)
         # In the client_mode, the dealer is the last node
         n = self.n if not client_mode else self.n+1
 
-        tag = f"{dealer_id}-{avss_id}-B-RBC"
+        tag = f"{dealer_id}-{avss_id}-B-AVID"
         send, recv = self.get_send(tag), self.subscribe_recv(tag)
 
-        # this will be replaced by Disperse
-        avss_msg = await reliablebroadcast(
-            tag,
-            self.my_id,
-            n,
-            self.t,
-            dealer_id,
-            broadcast_msg,
-            recv,
-            send
-        )
+        avid = AVID(n, self.t, dealer_id, recv, send, n)
```

Comment:
Is the `input_size_list` always going to be `n`?

Reply:
No, but i am using n now.